examples/cookbook/testing/integration/profiling/analysis_options.yaml

```diff
@@ -0,0 +1,9 @@
+# Take our settings from the example directory's main analysis_options.yaml
+# file. If necessary for a particular example, this file can also include
+# overrides for individual lints.
+
+include: ../../../../analysis_options.yaml
+
+linter:
+  rules:
+    avoid_print: false
```

Comment:
Any chance of removing dependence on `print`?

Reply:
thanks, well spotted, print is not even used in the sample